demos/multi_channel/object_detection_demo_yolov3/multichannel_object_detection_demo_yolov3_params.hpp

```diff
@@ -14,3 +14,88 @@ static const char thresh_output_message[] = "Optional. Probability threshold for
 /// \brief Flag to output raw scoring results<br>
 /// It is an optional parameter. Ignored for human-pose-estimation
 DEFINE_double(t, 0.5, thresh_output_message);
+
+std::vector<cv::Scalar> colors(80); 
+
+const char *coco_labels[80] = {
```

Comment:
You shouldn't hardcode the labels, because the user might want to use the demo with a model trained on a different dataset. Instead, if you want to have labels, add an option to load them from a file.